### ISSUE TITLE
Mobile/compact menu does not behave like other ‘hamburger menus’

### ISSUE THREAD

**Issue by ChrisXKeroack** [intel/openfl#846]
**Describe the bug**
Compact menu does not behave like a typical compact menu

**To Reproduce**
1. https://openfl.readthedocs.io/en/latest/index.html
2. Resize window to a narrow enough width to get the mobile/compact layout
3. Use keyboard, mouse and screenreader to interact with the 'hamburger menu' element

Notice:
a.	Functions as an expand/collapse for the mouse user
b.	Menu button isn’t keyboard accessible, doesn’t support aria-expanded
c.	Items in the sidebar are visually hidden but not programmatically hidden; keyboard user can tab to hidden / collapsed items

**Expected behavior**
Hamburger menu icon should present a menu that contains same menu items as are seen in desktop OR element should be represented as a simple show/hide for a collapsed sidebar, and when the sidebar is collapsed its elements shouldn't be able to take keyboard focus because they're not visible

**Screenshots**
![image](https://github.com/securefederatedai/openfl/assets/104644211/79cbc27a-5fc9-49e6-9ba4-c91ec1ad5911)

**Desktop:**
This issue is not OS or browser specific, but for tracking purposes, testing was done on Win11, in Edge Version 114.0.1823.58 (Official build) (64-bit), and the screenreader in use was the common 3rd party screenreader NVDA (http://www.nvaccess.org).

**Smartphone:**
This issue is not device, OS, or browser specific.

**Additional context**
This is an accessibility issue, please annotate this bug with the 'accessibility' label.

**Comment by fstrr** [comment]
This is an issue with the [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).

**Comment by theakshaypant** [comment]
Closing. As mentioned by @fstrr, this is an issue with [Read The Docs platform](https://github.com/readthedocs/readthedocs.org).
